docs/admin/privileges.rst

```diff
@@ -568,6 +575,33 @@ example if we revoke privileges on table ``doc.books`` from
 user ``wolfgang``, who is granted the role ``role_all_on_books``, also looses
 those privileges.
 
+.. hide:
+
+    cr> CREATE ROLE role_dml;
+    CREATE OK, 1 row affected (... sec)
+    cr> CREATE ROLE john;
+    CREATE OK, 1 row affected (... sec)
+
+If a user is granted the same privilege by inheriting two different roles, when
+revoking one of the roles, the user still keeps the privilege. For example if
+user ``john`` gets granted ```role_dql`` and ``role_dml``::
+
+    cr> GRANT DQL TO role_dql;
```

Comment:
I think this is the same as new note above?
Maybe we can drop a note since this one is superior, not only text but also practical example.